atd-vzv/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "atd-vzv",
-  "version": "1.40.0",
+  "version": "1.44.1",
```

Comment:
this version hasnt been updated in awhile

Reply:
whoops—i missed this when i approved the PR. i'm not sure what the best practice is, but i personally have never edited a package-lock file. i would just let this version get bumped whenever we get around to doing another `npm update --save`. but if y'all usually bump this version manually, then that's cool 👍

Reply:
alright i can remove this, i never know what the protocol is. but fwiw i didnt rly do it manually it was from when i ran `npm install`